osu.Game/OsuGame.cs

```diff
@@ -292,7 +292,7 @@ protected void LoadScore(ScoreInfo score, bool silent)
                 return;
             }
 
-            if ((screenStack.CurrentScreen as IOsuScreen)?.AllowExternalScreenChange != true)
+            if ((screenStack.CurrentScreen as IOsuScreen)?.DisallowExternalBeatmapRulesetChanges != false)
```

Comment:
This is not the same thing.

Reply:
Yeah, I realised this but the edge case behaviour doesn't matter either way.

Reply:
This has broken, e.g. while in player you can import a score and it will exit your play.